utils/build.ps1

```diff
@@ -1630,6 +1630,10 @@ function Build-Dispatch([Platform]$Platform, $Arch, [switch]$Test = $false) {
 }
 
 function Build-Foundation([Platform]$Platform, $Arch, [switch]$Test = $false) {
+  if ($Test) {
+    # Disable tests until we can build the new recore'd swift-foundation tests
+    return
+  }
```

Comment:
Any reason to not just modify the invoking script to not pass `foundation` to `-Test`? https://github.com/apple/swift/blob/main/utils/build-windows-toolchain.bat#L63

Reply:
Ah I can remove it there if you'd prefer if that'd catch all of the places that we try to invoke Foundation tests - do you think that's a better place to put it? (I hadn't taken a look at that file directly)

Reply:
I think that is a better place as it covers CI but still allows local developers to do "bad things" if they want.

Reply:
Sounds good, updated!